tests/examples/wibox/widget/progressbar/vertical.lua

```diff
@@ -3,16 +3,16 @@ local wibox  = require("wibox") --DOC_HIDE
 
 parent:add( --DOC_HIDE
 
-           wibox.widget {
-               {
-                   max_value     = 1,
-                   value         = 0.33,
-                   widget        = wibox.widget.progressbar,
-               },
-               forced_height = 100,
-               forced_width  = 20,
-               direction     = "east",
-               layout        = wibox.container.rotate,
-           }
+wibox.widget {
```

Comment:
Everything with DOC_HIDE isn't part of the example. It is removed before being added to the doc. In This case, as the indentation is uniform, it doesn't change the output, but in some other places it did.